src/openrct2/object/LargeSceneryObject.cpp

```diff
@@ -22,6 +22,7 @@
 
 #include <algorithm>
 #include <iterator>
+#include <nlohmann/json_fwd.hpp>
```

Comment:
Seems a bit suspicious to have the `fwd` on a source file, specially because there was no include before 🤔